src/components/truncate-long-text/truncateLongText.html

```diff
@@ -3,7 +3,11 @@
   styled directly in origin-web-common.  `truncated-content` is used by
   origin-web-console in certain contexts.
 -->
-<span ng-if="!truncated" ng-bind-html="content | highlightKeywords : keywords" class="truncated-content"></span>
+<span ng-if="!truncated">
+  <span ng-if="!linkify || (highlightKeywords | size)" ng-bind-html="content | highlightKeywords : keywords" class="truncated-content"></span>
+  <span ng-if="linkify && !(highlightKeywords | size)" ng-bind-html="content | linkify : '_blank'" class="truncated-content"></span>
+</span>
+<!-- To avoid truncating in middle of a link, we only apply linkify to expanded content -->
```

Comment:
> <!-- To avoid truncating in middle of a link, we only apply linkify to expanded content -->

<!-- To avoid truncating in middle of a link, we only optionally apply linkify to expanded content -->